sites/public/next.config.js

```diff
@@ -46,145 +45,15 @@ module.exports = withCSS(
             mapBoxToken: MAPBOX_TOKEN,
             housingCounselorServiceUrl: HOUSING_COUNSELOR_SERVICE_URL,
             gtmKey: process.env.GTM_KEY || null,
-            languages: process.env.LANGUAGES || "en",
             idleTimeout: process.env.IDLE_TIMEOUT,
           },
+          i18n: {
+            locales: process.env.LANGUAGES ? process.env.LANGUAGES.split(",") : ["en"],
+            defaultLocale: "en",
+          },
           sassLoaderOptions: {
             additionalData: tailwindVars,
           },
-          // exportPathMap adapted from https://github.com/zeit/next.js/blob/canary/examples/with-static-export/next.config.js
```

Comment:
Is all of this automatically generated now?

Reply:
Yep, that's the beauty of the updated system. No routing information needed in the config file whatsoever! 🤩

Reply:
@jaredcwhite will we be able to `hide` routes that we don't want live yet?